lib/engine/game/g_18_uruguay/step/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/buy_sell_par_shares'
+
+module Engine
+  module Game
+    module G18Uruguay
+      module Step
+        class BuySellParShares < Engine::Step::BuySellParShares
+          def can_buy?(entity, bundle)
+            return false if bundle.corporation == @game.rptla && @game.phase.name == '2'
```

Comment:
As you've added the availability of RPTLA shares to the phase's status, you should use it here:
```suggestion
            return false if bundle.corporation == @game.rptla && !@game.phase.status.include?('rptla_available')
```